title/arm9/source/main.cpp

```diff
@@ -866,10 +866,8 @@ int main(int argc, char **argv)
 		isDSiMode() ? fifoSendValue32(FIFO_USER_01, 10) : *(u32*)0x02000000 = 1;
 	}
 
-	if ((access(DSIMENUPP_INI, F_OK) != 0)
-	|| (ms().theme < 0) || (ms().theme == 3) || (ms().theme > 5)) {
-		// Create or modify "settings.ini"
-		(ms().theme == 3) ? ms().theme = 2 : ms().theme = 0;
+	if (access(DSIMENUPP_INI, F_OK) != 0) {
```

Comment:
Instead of resetting it outright, you could put it like this

```cpp
if ((access(DSIMENUPP_INI, F_OK) != 0) || (ms().theme < 0) || (ms().theme > 5) {
  ms().theme = 0;
  ms().saveSettings();
}
```